Whisky/Views/Bottle/Pins/PinView.swift

```diff
@@ -78,13 +78,14 @@ struct PinView: View {
         .sheet(isPresented: $showRenameSheet) {
             PinRenameView(name: $name)
         }
-        .onAppear {
+        .task {
             name = pin.name
-            Task.detached { @MainActor in
-                if let peFile = program.peFile {
-                    image = peFile.bestIcon()
-                }
+            guard let peFile = program.peFile else { return }
+            let task = Task<Image?, Never>.detached {
```

Comment:
Would removing `@MainActor` here not be sufficient? This seems a little verbose.